customs/extensions/_example/main.js

```diff
@@ -22,9 +22,10 @@ define([
     getMainView: function () {
       if (!this.mainView) {
         this.mainView = new MainView();
-        app.router.v.main.setView('#content', this.mainView);
       }
 
+      app.router.v.main.setView('#content', this.mainView);
```

Comment:
Does this fix a bug? let us know

Reply:
Yes, you can test the example, when you navigate to the extension page, it loads the first time, but when you go to another page (a table for example) and then try and go back to the extension page the `app.route.v.main.setView` never runs since `this.mainView` already exists, and so you just get a blank page.

Reply:
Can you send this separately?